nexuse2e-webapp/src/main/frontend/src/assets/i18n/en.json

```diff
@@ -102,7 +102,12 @@
     "minutes": "minutes",
     "idle": "idle",
     "noIdleConversations": "no idle conversations",
-    "idleSince": "Idle since"
+    "idleSince": "Idle since",
+    "certificates": "Certificates",
+    "noUsedCertificates": "no certificates in use",
+    "configuredFor": "Configured For",
+    "certificateName": "Certificate Name",
+    "timeRemaining": "Time Remaining"
```

Comment:
If "certificates" in `noUsedCertificates` is written in lower case, does "Remaining" needs also to be in lowe case?

Reply:
I think there is a different in our labels between small sentences and column titles.